gsettings.sh

```diff
@@ -26,7 +27,9 @@ list_proxy() {
          "$(gsettings get org.gnome.system.proxy.ftp port)"
     echo "${bold} socks ${normal} "\
          "$(gsettings get org.gnome.system.proxy.socks host) "\
-         "$(gsettings get org.gnome.system.proxy.socks port)"
+         "$(gsettings get org.gnome.system.proxy.socks port)"   
```

Comment:
Why has this line changed? 
Perhaps due to `tab` and `space`.

Reply:
I don't know.  Didn't change anything here. Maybe some intendation stuff from IDE?